sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Action.cs

```diff
@@ -2,24 +2,27 @@
 // Licensed under the MIT License.
 
 using System;
+using System.ComponentModel;
 
 namespace Azure.Security.KeyVault.Certificates
 {
     /// <summary>
     /// An action that will be executed.
     /// </summary>
-    public struct Action
+    public readonly struct Action : IEquatable<Action>
     {
-        private readonly string _value;
         internal const string AutoRenewValue = "AutoRenew";
         internal const string EmailContactsValue = "EmailContacts";
 
+        private readonly string _value;
+
         /// <summary>
-        /// Initializes a new instance of the Action struct with the specfied value.
+        /// Initializes a new instance of the <see cref="Action"/> structure.
         /// </summary>
-        public Action(string Action)
+        /// <param name="value"></param>
```

Comment:
> /// <param name="value"></param> [](start = 7, length = 33)

Need to add description for the param. This seems to be the case for all the edited enum like struct ctors.